doc/modules/neighbors.rst

```diff
@@ -427,6 +482,29 @@ and the ``'effective_metric_'`` is in the ``'VALID_METRICS'`` list of
 same order as the number of training points, and that ``leaf_size`` is 
 close to its default value of ``30``.
 
+Valid Metrics for Nearest Neighbor Algorithms
+---------------------------------------------
```

Comment:
I am not sure how to do that :\ (doctest showing `valid_metrics` class attributes) and searching didn't help. Can you point me to an example?